.github/workflows/gradle-test.yml

```
@@ -25,3 +25,9 @@ jobs:
 
     - name: Run Gradle test
       run: ./gradlew test
+
+    - name: Run Gradle javadocs
+      run: ./gradlew javadocs
```

I think according to the build failure...

```suggestion
    - name: Run Gradle javadoc
      run: ./gradlew javadoc
```